api/v1alpha4/types.go

```diff
@@ -116,6 +116,38 @@ type SubnetFilter struct {
 	NotTagsAny      string `json:"notTagsAny,omitempty"`
 }
 
+type PortOpts struct {
```

Comment:
as follow up, we may need consider add some samples in how to use this opts (not this PR)

Reply:
I can certainly add additional documentation in a follow up PR.